docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -146,7 +146,7 @@ <h2>The result</h2>
 
 					camera.position.z = 5;
 
-					var animate = function () {
+					var animate =  () =&gt; {
```

Comment:
For simplicity, I vote for the more common:
```js
function animate() {
}
```

Reply:
I agree. especially within the introductory section.